lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -44,7 +44,7 @@ def check_max_version(name, maximal_version)
         end
       end
 
-      def downstream_installation?
+      def satellite_installation?
```

Comment:
This should be moved to the satellite feature and should be using feature(:package_manager) instead of direct usage of rpm